sparkpost/transmissions.py

```diff
@@ -265,15 +266,18 @@ def get(self, transmission_id):
         results = self._fetch_get(transmission_id)
         return results['transmission']
 
-    def list(self):
+    def list(self, **kwargs):
         """
         Get a list of your transmissions
 
+        :param campaign_id: ID of the campaign used by the transmissions
+        :param template_id: ID of the template used by the transmissions
+
         :returns: list of transmissions
         :raises: :exc:`SparkPostAPIException` if API call fails
         """
-        results = self.request('GET', self.uri)
-        return results
+        warnings.warn('deprecated', DeprecationWarning)
```

Comment:
I would customize the message rather than put in deprecated. For instance we could say it will be removed from the API soon. After looking a little more, you might want to use `PendingDeprecationWarning`. Curious - what does this look like at the command line for users?